content/docs/use-cases/data-registry/tutorial.md

```diff
@@ -166,7 +133,7 @@ scenarios such as a _model zoo_.
 
 See also the `dvc.api.read()` and `dvc.api.get_url()` functions.
 
-## Updating registries
+## Maintaining registries
```

Comment:
Updating was better (and anyways, let's not change minot things please in an allready long PR)

Reply:
I forgot I changed that (to match the intro sentence).

```suggestion
## Updating registries
```